.github/docs_pr_change_template.md

```diff
@@ -0,0 +1,12 @@
+## Description
+
+<!-- Describe the changes made in this PR -->
+
+## Checklist
+- [ ] This change matches the [docs style guide](https://goteleport.com/docs/contributing/documentation/style-guide/)
+- [ ] I viewed the changes on the dev server (`yarn dev`) to confirm they render without error
+- [ ] The terms used in this change correspond to those in [Teleport Core Concepts](https://goteleport.com/docs/core-concepts)
```

Comment:
It would be nice to be able to lint for this one using gravitational/docs#238, though we still might need to keep this list item here. What do you think?

Reply:
I think if that PR gets merged we should definitely remove this line.